pkg/daemon/command/command.go

```diff
@@ -26,14 +26,15 @@ type DaemonCommand struct {
 	Upgrade        bool   `type:"flag" name:"upgrade" default:""`
 	ThreadNum      string `type:"param" name:"thread-num"`
 	// `--id` is required by `--supervisor` when starting nydusd
-	ID         string `type:"param" name:"id"`
-	Config     string `type:"param" name:"config"`
-	Bootstrap  string `type:"param" name:"bootstrap"`
-	Mountpoint string `type:"param" name:"mountpoint"`
-	APISock    string `type:"param" name:"apisock"`
-	LogLevel   string `type:"param" name:"log-level"`
-	Supervisor string `type:"param" name:"supervisor"`
-	LogFile    string `type:"param" name:"log-file"`
+	ID           string `type:"param" name:"id"`
+	Config       string `type:"param" name:"config"`
+	ConfigSource string `type:"param" name:"config-source"`
```

Comment:
Does nydusd already have the CLI argument `config-source` ?

Reply:
I made modifications to nydusd and submitted PR to add this CLI parameter